<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tests</class>
 <widget class="QWidget" name="Tests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pageName_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Benchmark files</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>← Go back</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NCounter">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>845</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>845</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>N =</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testAllButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>835</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test all</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Vector</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="vectorTable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>442</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adding</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Searching</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="textFileTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>442</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adding</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Reading</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Searching</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Text File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Binary File</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="binaryFileTable">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>160</y>
     <width>442</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adding</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Reading</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Searching</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Σ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="testTextFileButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>740</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testVectorButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>740</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testBinaryFileButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>740</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
